install/kubernetes/helm/istio-init/templates/job-crd-10.yaml

```diff
@@ -14,6 +14,12 @@ spec:
       - name: istio-init-crd-10
         image: "{{ .Values.global.hub }}/kubectl:{{ .Values.global.tag }}"
         imagePullPolicy: {{ .Values.global.imagePullPolicy }}
+{{- if .Values.job.resources }}
+        resources:
+{{ toYaml .Values.job.resources | indent 10 }}
+{{- else }}
+        resources: {}
```

Comment:
default value for resources is empty, isn't it? better to remove the else block.